tools/cldr-apps/src/main/webapp/js/CldrSurveyVettingLoader.js

```diff
@@ -1237,10 +1257,10 @@ function showV() {
 				 * Show the "possible problems" section which has errors for the locale
 				 */
 				function showPossibleProblems(flipper, flipPage, loc, session, effectiveCov, requiredCov) {
-					surveyCurrentLocale = loc;
-					require(["dojo/ready"], function(ready) {
-						ready(function() {
-							var url = cldrStatus.getContextPath() + "/SurveyAjax?what=possibleProblems&_=" + surveyCurrentLocale + "&s=" + session + "&eff=" + effectiveCov + "&req=" + requiredCov + cacheKill();
+					cldrStatus.setCurrentLocale(loc);
+					/// require(["dojo/ready"], function(ready) {
+					/// 	ready(function() {
+							var url = cldrStatus.getContextPath() + "/SurveyAjax?what=possibleProblems&_=" + cldrStatus.getCurrentLocale() + "&s=" + session + "&eff=" + effectiveCov + "&req=" + requiredCov + cacheKill();
```

Comment:
"ready" seems superfluous here, since showV -- this entire file -- is now only called by cldrGui.run which already depends on dojo/ready... but this needs more testing, and then delete the commented-out lines and reformat